### ISSUE TITLE
aFED balance doesn't refresh after withdraw harvest tokens

### ISSUE THREAD

**Issue by UmckaBear** [dFed-finance/interface#12]
aFED balance on top right corner will refresh only in case of full page refresh or reconnecting wallet.
If you are using Harvest option and withdraw aFED tokens, balance will not refresh.
![Balance Don't refresh](https://user-images.githubusercontent.com/31537952/101284562-420d2f00-37e9-11eb-96f2-a3386259b98f.png)

**Comment by UmckaBear** [comment]
Same situation when you disconnect wallet. On screenshot wallet is disconnected but aFED balance still shown
![Balance Don't refresh 2](https://user-images.githubusercontent.com/31537952/101289781-3892bf80-3807-11eb-8122-83f6b7ae80d0.png)

**Comment by shawnying** [comment]
Manually refresh is required. Automatic refreshment is not supported yet.

**Comment by UmckaBear** [comment]
> Manually refresh is required. Automatic refreshment is not supported yet.

Yes. You can only manually refresh page and then get balance refreshment. That's why it was reported as UI bug.
